### DISCUSSION TITLE
SPL results understanding

### DISCUSSION THREAD

**Discussion by fredmzlp** [Ifsttar/I-Simpa#355] [Q&A]
Hi all,
I'm using I-Simpa to simulate and analyse SPL distribution in a room. There are a few items I'm unsure about, and I would appreciate your help in clarifying them.

My understanding of "global" and "total" sound level results:
When reading SPL dB at a point receiver, by opening the "sound level" tab, the "global" value, I understand is the sum of all frequency bands (say, broadband sound level).
On the other hand, the "total" value (column at the right end of table) is the sum of all time steps values per frequency. 
Opening the "acoustic parameters" tab, the table gives as "sound level dB", these "total" values.

I assume there is a direct parallel to surface receiver results for "instantaneous sound level", showing each time step of the response, and "cumulating sound level", showing the "total" values. Is this correct?

A few questions:
   a) I am wondering what would be the use of the "total"sound level  results...? It does not represent any standardized noise exposure metric It seems. It is also a value that is never registered by a receiver at any time. 

   b) What is the duration of the sound source emitted? Is it one sample long (impulse)?

   c) Is there a way of having max sound levels mapped in a surface receiver? This could give me an idea of the distribution of  pressure throughout that is actually registered at a given location due to an impulse event.

   d) Is there a way to have on the surface and point receivers the default sound level results in db(A)?

Thank you all for your collective wisdom.
Best,
   - Fred

**Comment by nicolas-f** [comment]
Hi,

Thanks for your interest in I-Simpa.

a) What I understood but @Picaut can correct me. As SPPS is emitting an impulse, we have to use the cumulative noise level in order to get the equivalent of a steady noise level source.
https://i-simpa-wiki.readthedocs.io/fr/latest/code_configuration_SPPS.html#sound-level-punctual-receier

b) There is no duration as it is an impulse. The duration in SPPS parameters is simulation length, described here:
https://i-simpa-wiki.readthedocs.io/fr/latest/code_configuration_SPPS.html#spps-calculation-parameters

c) There is no tool to search for max level. And do not use the surface map values as it is not comparable with punctual receivers (the purpose of surface receivers are only for display and are not part of the validation of the computation model)

d) There is no such option in the software

best regards,

-Nicolas

**Reply by fredmzlp** [reply]
Hi Nicolas,

Thank you for you answers, that's great! 
Much appreciated.

Best,
- Frederico
